### PR TITLE
Add PowerShell snippets for postinstall

### PR DESCRIPTION
**Pull request by tats-u**
https://github.com/python-poetry/install.python-poetry.org/issues/66

This PR allows Windows users to activate `poery` command without forcing them to google how to do it.

### CODE REVIEW

**Comment by tats-u** [comment]
`````
Poetry (1.1.14) is installed now. Great!
To get started you need Poetry's bin directory (C:\Users\runneradmin\AppData\Roaming\Python\Scripts) in your `PATH`
environment variable.
You can choose and execute one of the following commands in PowerShell:
A. Appends the bin directory to your user environment variable `PATH`:
```
[Environment]::SetEnvironmentVariable("Path", [Environment]::GetEnvironmentVariable("Path", "User") + ";C:\Users\runneradmin\AppData\Roaming\Python\Scripts", "User")
```
B. Tries to appends the bin directory to PATH every when you run PowerShell (>=6 recommended):
```
echo 'if (-not (Get-Command poetry -ErrorAction Ignore)) { $env:Path += ";C:\Users\runneradmin\AppData\Roaming\Python\Scripts" }' | Out-File -Append $PROFILE
```
Alternatively, you can call Poetry explicitly with `C:\Users\runneradmin\AppData\Roaming\Python\Scripts\poetry`.
You can test that everything is set up by executing:
`poetry --version`
`````

**Comment by johnthagen** [comment]
For reviewers, here is a useful StackOverflow answer on the subject: https://stackoverflow.com/a/257120

**Review by johnthagen — CHANGES_REQUESTED** [review]
[Disclaimer, not a Poetry dev]

First off, thank you for making this PR. I think this is an important step to easing new Windows users into Poetry.

My personal feedback is that we should try to keep this simpler. The installer already prints quite a lot of text and having multiple Powershell commands I think could overwhelm the target audience.

If we look at the Unix variant of this for inspiration, it is very simple:

```py
POST_MESSAGE_CONFIGURE_UNIX = """
Add `export PATH="{poetry_home_bin}:$PATH"` to your shell configuration file.
"""
```

I would recommend we shorten this message to:

```py
POST_MESSAGE_CONFIGURE_WINDOWS = """
Execute the following in Powershell:

[Environment]::SetEnvironmentVariable("Path", [Environment]::GetEnvironmentVariable("Path", "User") + ";{poetry_home_bin}", "User")
"""
```

**Comment by johnthagen** [comment]
I tested Option A in this MR in a clean Windows 10 VM and it worked as expected. 🚀

**Comment by tats-u** [comment]
Unix OSes have no choice other than editing profile files for their shells, so how to activate Poetry can be kept simple.

In contrast of this, Windows has the built-in manager of environment variables.
PowerShell and CMD have profiles like shells for Unix.
This is why 2 option can be provided for Windows.

**Comment by johnthagen** [comment]
@neersighted Would you be willing to review this PR? I personally think it would be important before the upcoming 1.2 final release. ❤️

**Review by abn — CHANGES_REQUESTED** [review]

File: install-poetry.py [hunk 1]
```diff
@@ -269,7 +269,21 @@ def _get_win_folder_with_ctypes(csidl_name):
...
+POST_MESSAGE_CONFIGURE_WINDOWS = """
+You can choose and execute one of the following commands in PowerShell:
+
+A. Appends the bin directory to your user environment variable `PATH`:
+
+```
+[Environment]::SetEnvironmentVariable("Path", [Environment]::GetEnvironmentVariable("Path", "User") + ";{poetry_home_bin}", "User")
+```
+
+B. Tries to appends the bin directory to PATH every when you run PowerShell (>=6 recommended):
```
[the remark this answers is not in the document]
Comment (abn) [reply install-poetry.py:281]: @tats-u think you missed this one :)

**Pull request merged**